tests/components/events/view/SingleEventCard.test.js

```diff
@@ -0,0 +1,39 @@
+/* global React :true */
+/* global mount :true */
+/* global describe :true */
+/* global it :true */
+/* global expect :true */
+/* eslint no-undef: "error" */
+
+import SingleEventCard from '../../../../src/components/events/view/SingleEventCard';
+
+function setup(path){
+    const pathName = path;
+    const event = {
+        name: 'testEvent',
+        description: 'event description',
+        category: 'testin',
+        location: 'testPlace',
+        event_date : 'Thu, 03 May 2018 00:00:00 GMT',
+        orgarniser : 'tester'
+    };
+    return mount(
+        <SingleEventCard event={event} pathName={pathName} />
+    );
+};
+
+describe('SingleEventCard component', ()=> {
+    it('Should display the passed event in appropriate fields',()=> {
+        const wrapper = setup('/events/myEvents');
```

Comment:
Why not use the `beforeEach()` function to setup the wrapper. then `unmount` in the `afterEach()`

Reply:
It's a nice and easier way, However, in this case, I opted for this in order to enable passing different `props` and testing how the component will behave given a certain set of props, in this case, the path.